### DISCUSSION TITLE
API Documentation

### DISCUSSION THREAD

**Discussion by stevespaw** [dyc3/opentogethertube#1483] [Q&A]
Is there a document describing the API and what features can be accessed. Authorization details?

**Comment by dyc3** [comment]
There is an openapi spec here: https://github.com/dyc3/opentogethertube/blob/master/docs/api.yaml

**Comment by stevespaw** [comment]
Ok thanks, a VERY quick skim through. Is there Play, Pause, Skip, Next and Previous ?  In the API?

**Reply by dyc3** [reply]
Not by a normal API request, you would have to connect via websocket.

**Reply by stevespaw** [reply]
Do you have any documentation for the websocket connection?

**Reply by dyc3** [reply]
Not really. The closest thing I have are the type definitions for the messages. https://github.com/dyc3/opentogethertube/blob/master/common/models/messages.ts
